src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -198,6 +201,11 @@ private BoundExpression BindArgListOperator(InvocationExpressionSyntax node, Dia
                 else if (argument.Type.IsVoidType())
                 {
                     Error(diagnostics, ErrorCode.ERR_CantUseVoidInArglist, argument.Syntax);
+                    wasError = true;
+                }
+                else if (analyzedArguments.RefKind(i) == RefKind.None)
+                {
+                    analyzedArguments.Arguments[i] = BindToNaturalType(analyzedArguments.Arguments[i], diagnostics);
```

Comment:
Do we want to set HasErrors if this produces errors as well?

Reply:
Since the error flag is aggregated up the bound nodes, that will happen automatically.

---
In reply to: [301337777](https://github.com/dotnet/roslyn/pull/37052#discussion_r301337777) [](ancestors = 301337777)